InverterApp/LcdDisplay.ino

```diff
@@ -0,0 +1,224 @@
+#include <Wire.h>
+#include <LiquidCrystal_I2C.h> 
+#include <TM1637TinyDisplay.h>
+#include "gpioHandler.h"
+
+
+//Created new object that connected to the Arduino board.
+LiquidCrystal_I2C lcd(0x27,20,4);     
+//The byte arrays were created using the website https://maxpromer.github.io/LCD-Character-Creator/
+byte fullBox[] = 
+{
+  B11111,
+  B11111,
+  B11111,
+  B11111,
+  B11111,
+  B11111,
+  B11111,
+  B11111
+};
+byte halfBoxBottom[] =
+{
+  B00000,
+  B00000,
+  B00000,
+  B00000,
+  B11111,
+  B11111,
+  B11111,
+  B11111  
+};
+byte halfBoxTop[] =
+{
+  B11111,
+  B11111,
+  B11111,
+  B11111,
+  B00000,
+  B00000,
+  B00000,
+  B00000  
+};
+
+byte topLeftAChar[] = {
+  B00001,
+  B00011,
+  B00111,
+  B01111,
+  B01111,
+  B11111,
+  B11111,
+  B11111
+};
+byte sideLeftAChar[] = {
+  B00001,
+  B00011,
+  B00111,
+  B01111,
+  B11111,
+  B11111,
+  B11111,
+  B11111
+};
+byte sideLeftAInnerChar[] = {
+  B11111,
+  B11111,
+  B11110,
+  B11110,
+  B11100,
+  B11000,
+  B10000,
+  B10000
+};
+byte sideLeftInnerCharConnector[] = {
+  B11111,
+  B11111,
+  B11110,
+  B11100,
+  B11111,
+  B11111,
+  B11111,
+  B11111
+};
+byte topRightSCurve[] = {
+  B10000,
+  B11000,
+  B11100,
+  B11110,
+  B11111,
+  B11111,
+  B11111,
+  B11111
+};
+byte topLeftSCurve[] = {
+  B00001,
+  B00011,
+  B00111,
+  B01111,
+  B11111,
+  B11111,
+  B11111,
+  B11111
+};
+byte bottomLeftSCurve[] = {
+  B00000,
+  B00000,
+  B11100,
+  B11110,
+  B11110,
+  B01111,
+  B01111,
+  B00111
+};
+
+void setup()
+
+{       
+  GPIOHandler gpioMan;
+  gpioMan.Init();
+   
+  //Initialized screen and backlight.
+  lcd.init();
+  lcd.backlight();
+  
+  //Characters used to create the logo
+  lcd.begin(20, 4);
+  lcd.createChar(0, fullBox);
+  lcd.createChar(1, halfBoxBottom);
+  lcd.createChar(2, halfBoxTop);
+  lcd.createChar(3, topLeftAChar);
+  lcd.createChar(4, sideLeftAChar);
+  lcd.createChar(5, sideLeftAInnerChar);
+  lcd.createChar(6, sideLeftInnerCharConnector);
+  lcd.createChar(7, topRightSCurve);
```

Comment:
I would create a function in `gpioHandler` called `LCDInit` which does all this.